.github/workflows/test.yaml

```diff
@@ -0,0 +1,57 @@
+name: Unit Tests
```

Comment:
Thanks! Let's remove the Travis config then

Reply:
Note that `actions/setup-go` currently does not support gotip, though it should be easier to add now that they have the necessary infrastructure in [actions/go-versions](https://github.com/actions/go-versions).

Reply:
I’ve added codecov upload step in GitHub Actions, but I think we should keep Travis for `gotip` tests for now.